modules/ossm-add-project-using-label-selectors-cli.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-label-selectors-cli_{context}"]
+= Adding a project to the {SMProductShortName} using label selectors with the CLI 
+
+You can use labels selectors to add a project to the {SMProductShortName} with the CLI.
+
+.Prerequisites
```

Comment:
```suggestion
.Prerequisites
```
- These 3 list items need to have parallel grammatical constructions -- either all complete sentences or all sentence fragments.